tool/conda/singa/meta.yaml

```diff
@@ -50,7 +50,7 @@ requirements:
     - protobuf 3.10.0         # [osx]
     - protobuf 3.9.2          # [linux]
     - glog 0.3.5
-    - numpy 1.16.5
```

Comment:
I think we can use a range like >=1.16,<2.0

Reply:
Yes, updated the code. The travis CI build the conda package sucessfully 
https://travis-ci.org/github/apache/singa/jobs/703751043#L2518